netbench/cdk/src/main/java/com/aws/ClientServerStack.java

```diff
@@ -0,0 +1,115 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.aws;
+
+import software.constructs.Construct;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.s3.Bucket;
+import software.amazon.awscdk.services.ecs.Cluster;
+import software.amazon.awscdk.services.ec2.Vpc;
+import software.amazon.awscdk.services.ecs.*;
+import software.amazon.awscdk.services.autoscaling.AutoScalingGroup;
+import software.amazon.awscdk.services.ec2.InstanceType;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpoint;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpointOptions;
+import software.amazon.awscdk.services.ec2.GatewayVpcEndpointAwsService;
+import software.amazon.awscdk.services.ec2.BastionHostLinux;
+import software.amazon.awscdk.services.ec2.*;
+import software.amazon.awscdk.services.ssm.StringParameter;
+
+
+import software.amazon.awscdk.services.apigateway.LambdaRestApi;
+import software.amazon.awscdk.services.lambda.Code;
+import software.amazon.awscdk.services.lambda.Function;
+import software.amazon.awscdk.services.lambda.Runtime;
+import software.amazon.awscdk.RemovalPolicy;
+
+import java.util.HashMap;
```

Comment:
Do you need these? They don't seem used even by the commented out code? same with `software.amazon.awscdk.StackProps`

Reply:
I was using them for some testing constructs that I left out of the commit